### PR TITLE
Core 610

### PR DESCRIPTION
**Pull request by hasahmad**

### CODE REVIEW

File: src/@commonsku/styles/Toggle.tsx [hunk 1]
```diff
@@ -100,19 +100,25 @@ const ToggleLink = styled.a<{selected?: boolean, stretch?:boolean, size?: Toggle
...
   }`
 
-type ToggleProps = React.PropsWithChildren<{
+type ToggleProps = {
   stretch?: boolean;
   onClick?: React.MouseEventHandler<HTMLDivElement>;
   size?: ToggleSize;
   style?: React.CSSProperties;
-} & SharedStyleTypes>;
+  children?: (string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>>)[];
```
Comment (robert-commonsku) [inline src/@commonsku/styles/Toggle.tsx:108]: I think you can use `children: React.ReactNode` for the `children` prop.   I think it's defined as
```
type ReactNode =
  | ReactElement
  | string
  | number
  | Iterable<ReactNode>
  | ReactPortal
  | boolean
  | null
  | undefined;

**Review by robert-commonsku — APPROVED** [review]
Might want to add the `AnnounceIcon` to App.tsx

File: src/@commonsku/utils/utils.ts [hunk 2]
```diff
@@ -11,3 +11,11 @@ export function isClientSide() {
 export function toArray<T>(x: T | T[]): T[] {
   return Array.isArray(x) ? x : [x];
 }
+
+export function isIterable(input: any): input is Iterable<any> {
```
Comment (robert-commonsku) [inline src/@commonsku/utils/utils.ts:15]: Where is this being used?

**Review by robert-commonsku — DISMISSED** [review]

File: src/@commonsku/utils/utils.ts (hunk 2 above)
Comment (hasahmad) [reply src/@commonsku/utils/utils.ts]: updated

**Review by robert-commonsku — APPROVED** [review]

**Pull request merged**